tensorrt_llm/_torch/models/modeling_deepseekv3.py

```
@@ -1028,25 +1075,34 @@ def forward(
         position_ids: Optional[torch.IntTensor] = None,
         inputs_embeds: Optional[torch.FloatTensor] = None,
     ) -> torch.Tensor:
+        print(f"[DEBUG] DeepSeekV3Model.forward - input_ids shape: {input_ids.shape if input_ids is not None else None}")
```

Ruff (0.12.2)

Line too long (121 > 120)

(E501)